tests/spec/valid/scales-bin.ts

```diff
@@ -83,7 +83,7 @@ const spec: Spec = {
 
   "axes": [
     {"orient": "bottom", "scale": "xscale", "tickCount": 20, "zindex": 1},
-    {"orient": "left", "scale": "yscale", "tickCount": 5, "zindex": 1}
+    {"orient": "left", "scale": "yscale", "tickCount": "hour", "zindex": 1}

```

Comment:
Did the original vega example change? If not, let's leave the examples as they are.

Reply:
So where should I add a test for this?

Reply:
It's okay without a test in this case. We'll add one as we add more example specs.